src/components/molecules/main-nav/main-nav.tsx

```diff
@@ -57,12 +60,10 @@ const MainNav = ({ navList }: MainNavProps) => {
 
             <DrawerBody>
               <ul>
-                {navList.map((navItem, index) => (
+                {mobileList.map((navItem, index) => (
```

Comment:
```suggestion
                {navList.map((navItem, index) => (
```
Please remove this instance of mobileList.